src/api/spec/models/notification_rss_feed_item.rb

```diff
@@ -0,0 +1,83 @@
+require "rails_helper"
+
+RSpec.describe Notification::RssFeedItem do
+  describe '.cleanup' do
+    let(:user) { create(:confirmed_user, login: 'manolo') }
```

Comment:
The login is defined as a sequence in the user factory so you don't need to specify login here. Unless you really want him to be called Manolo lol..

https://github.com/evanrolfe/open-build-service/blob/master/src/api/spec/factories/users.rb#L5